src/gui/vector/qgsfieldcalculator.cpp

```diff
@@ -338,8 +351,15 @@ void QgsFieldCalculator::accept()
     }
 
     mVectorLayer->endEditCommand();
+    if ( mNewFieldGroupBox->isChecked() )
+    {
+      pushMessage( tr( "Field %1 created successfully" ).arg( mVectorLayer->name() ) );
```

Comment:
this doesn't seem right: isn't it better to show the field name instead of the layer name? (same below)